deployment/network-operator/README.md

```diff
@@ -0,0 +1,163 @@
+$ helm install -n network-operator --create-namespace --wait mellanox/network-operator network-operator
+
+# View deployed resources
+$ kubectl -n network-operator get pods
+```
+
+>__Note:__ By default the operator is deployed without an instance of `NicClusterPolicy`
+custom resource. The user is required to create it later with configuration matching the cluster or use
+chart parameters to deploy it together with the operator.
+
+## Chart parameters
+
+In order to tailor the deployment of the network operator to your cluster needs
+We have introduced the following Chart parameters.
+
+### General parameters
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `nfd.enabled` | bool | `True` | deploy Node Feature Discovery |
+| `operator.repository` | string | `mellanox` | Network Operator image repository |
+| `operator.image` | string | `network-operator` | Network Operator image name |
+| `operator.tag` | string | `None` | Network Operator image tag |
+| `deployCR` | bool | `false` | Deploy `NicClusterPolicy` custom resource according to provided parameters |
+
+### Custom resource parameters
+
+#### Mellanox OFED driver
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `ofedDriver.deploy` | bool | `false` | deploy Mellanox OFED driver container |
+| `ofedDriver.repository` | string | `mellanox` | Mellanox OFED driver image repository |
+| `ofedDriver.image` | string | `ofed-driver` | Mellanox OFED driver image name  |
+| `ofedDriver.version` | string | `5.0-2.1.8.0` | Mellanox OFED driver version  |
+
+#### NVIDIA Peer memory driver
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `nvPeerDriver.deploy` | bool | `false` | deploy NVIDIA Peer memory driver container |
+| `nvPeerDriver.repository` | string | `mellanox` | NVIDIA Peer memory driver image repository |
+| `nvPeerDriver.image` | string | `nv-peer-mem-driver` | NVIDIA Peer memory driver image name  |
+| `nvPeerDriver.version` | string | `1.0-9` | Mellanox OFED driver version  |
+| `nvPeerDriver.gpuDriverSourcePath` | string | `/run/nvidia/driver` | GPU driver soruces root filesystem path(usually used in tandem with [gpu-operator](https://github.com/NVIDIA/gpu-operator)) |
+
+#### RDMA Device Plugin
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `devicePlugin.deploy` | bool | `true` | Deploy device plugin  |
+| `devicePlugin.repository` | string | `mellanox` | Device plugin image repository |
+| `devicePlugin.image` | string | `k8s-rdma-shared-dev-plugin` | Device plugin image name  |
+| `devicePlugin.version` | string | `v1.0.0` | Device plugin version  |
+| `devicePlugin.resources` | list | See below | Device plugin resources |
+
+##### RDMA Device Plugin Resource configurations
+
+Consists of a list of RDMA resources each with a name a a list of RDMA capable network devices
+to be associated with the resource.
+
+```
+resources:
```

Comment:
what about the selector config? 
Do we need additional work to support it in the network operator ?

Reply:
at the time this was written, selector was not merged.
we will need:
1. make a release of k8s-rdma-shared-device-plugin
2. modify helm chart to support it and update this README